<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibDialog</class>
 <widget class="QDialog" name="CalibDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calibration</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>340</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="lidar_type_box">
   <item>
    <property name="text">
     <string>Jaguar</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Falcon</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="lidar_field_box">
   <item>
    <property name="text">
     <string>intensity</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>reflectance</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lidar_type_label">
   <property name="text">
    <string>LidarModel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lidar_field_label">
   <property name="text">
    <string>ScalarField</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CalibDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CalibDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
